src/daemon/ca/certauth.rs

```diff
@@ -1059,16 +1083,15 @@ impl CertAuth {
     ///
     /// Note that when we receive the updated certificate, we will republish
```

Comment:
Is this comment about republishing still correct given the removed publication actions referred to in comments above?